alf/environments/suite_bsuite.py

```diff
@@ -0,0 +1,106 @@
+import alf
+from alf.environments import gym_wrappers, alf_wrappers, alf_gym_wrapper
+from alf.environments.suite_gym import wrap_env
+
+import bsuite
+from bsuite import sweep
+from bsuite.utils import gym_wrapper
+from gym import spaces
+from dm_env import specs
+import numpy as np
+from typing import Any, Dict, Tuple
+
+
+@alf.configurable
+def load(environment=sweep.CARTPOLE_SWINGUP[0],
+         env_id=None,
+         discount=1.0,
+         max_episode_steps=None,
+         gym_env_wrappers=(),
+         alf_env_wrappers=()):
+    """Loads the selected environment and wraps it with the specified wrappers.
+
+    Note that by default a TimeLimit wrapper is used in wrap_env to limit episode 
+    lengths to the default benchmarks defined by the registered environments.
+
+    Args:
+        environment_name (str): Name for the environment to load.
+        env_id (int): (optional) ID of the environment.
+        discount (float): Discount to use for the environment.
+        max_episode_steps (int): If None the max_episode_steps will be set to zero as not
```

Comment:
The description of this argument has an incorrect format.

Reply:
Changed environment parameter to environment_name and reformatted the max_episode_steps description.